src/components/AppHelp.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+	<div>
+		<AppSettingsDialog :open.sync="settingsOpen" :showNavigation="true">
+			<div class="exit icon-close" @click="settingsOpen = false; "/>
+			<h2>{{ t('notes', 'Notes Application') }}</h2>
+			<AppSettingsSection :title="t('notes', 'Basics')">
+				<div class="feature icon-add">
+					{{ t('notes', 'Start writing a note by clicking on “{newnote}” in the app navigation.', { newnote: t('notes', 'New note') }) }}
+				</div>
+				<div class="feature icon-fullscreen">
+					{{ t('notes', 'Write down your thoughts without any distractions.') }}
+				</div>
+				<div class="feature icon-files-dark">
+					{{ t('notes', 'Organize your notes in categories.') }}
+				</div>
+				<button class="button-icon-add icon-add" @click="onNewNote">
+					{{ t('notes', 'Create a sample note with markdown') }}
+				</button>
+			</AppSettingsSection>
+			<AppSettingsSection :title="t('notes', 'Markdown')">
+				<div class="feature icon-toggle-filelist">
+					{{ t('notes', 'Use Markdown markups to style your text.') }}
+				</div>
+				<button @click="onNewNote">
```

Comment:
The button should look like the similar button above. Or should we remove this second button at all?
```suggestion
				<button class="button-icon-add icon-add" @click="onNewNote">
```

Reply:
Good catch!